src/lib/config.ts

```diff
@@ -1,10 +1,38 @@
 import * as _ from 'lodash';
+import * as memoizee from 'memoizee';
+
+const openVpnConfig = `
+client
+remote VPN_DETAILS
+resolv-retry infinite
+
+remote-cert-tls server
+ca /etc/openvpn/ca.crt
+auth-user-pass /var/volatile/vpn-auth
+auth-retry none
+script-security 2
+up /etc/openvpn-misc/upscript.sh
+up-restart
+down /etc/openvpn-misc/downscript.sh
+
+comp-lzo
+dev resin-vpn
+dev-type tun
+proto tcp
+nobind
+
+persist-key
+persist-tun
+verb 3
+user openvpn
+group openvpn
+`;
 
 export const requiredVar = (varName: string): string => {
 	const s = process.env[varName];
 	if (s == null) {
 		console.error(`Missing environment variable: ${varName}`);
-		process.exit(1);
+		process.exitCode = 1;
```

Comment:
If we are changing the `exit()` call, then I think it would be better to throw the missing environment variable error instead of the `Unreachable` error. This looks like we should expect an error, followed by a random "Unreachable" exception message.